lib/OpenQA/WebAPI/Auth/OpenID.pm

```diff
@@ -103,14 +103,19 @@ sub auth_response ($c) {
 
     my $err_handler = sub ($err, $txt) {
         $c->app->log->error("OpenID: $err: $txt");
-        $c->flash(error => "$err: $txt");
+        $c->flash(error =>
+"$err: $txt. Please retry again. If this reproduces please report the problem to the system administrators."
```

Comment:
So this will spill out "Failed to login: OpenID provider '$op_uri' returned invalid data on a login attempt. Report to the authentication server administrators. Please retry again. If this reproduces please report the problem to the system administrators."?

Reply:
uh, nope, that's not what I wanted :) Please see the updated version.